e2e/test/scenarios/admin/settings/sso/saml.cy.spec.js

```diff
@@ -76,6 +76,31 @@ describeEE("scenarios > admin > settings > SSO > SAML", () => {
     getSamlCard().findByText("Set up").should("exist");
   });
 
+  it("should display a message when using env variable values", () => {
```

Comment:
You can convert it to a unit test, there is already the setup code for that:
`frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/sso-saml.unit.spec.tsx`